rfc6962-bis.xml

```diff
@@ -1412,21 +1424,39 @@ but it is expected there will be a variety.
+                  <t>Once verified, remove the subjectAltName extension from the TBSCertificate.</t>
+                </list>
               </t>
-              <t>
-                Replace leftmost labels of each DNS-ID with <spanx style="verb">?</spanx>, based on the INTEGER value corresponding to that DNS-ID in the extension.
+            </list>
+          </t>
+          <t>
+            If the SCT checked is for a Precertificate (where the <spanx style="verb">type</spanx> of the <spanx style="verb">TransItem</spanx> is <spanx style="verb">precert_sct_v2</spanx>), then the client SHOULD also remove embedded v1 SCTs, identified by OID 1.3.6.1.4.1.11129.2.4.2 (See Section 3.3. of <xref target="RFC6962"/>), in the process of reconstructing the TBSCertificate. That is to allow embedded v1 and v2 SCTs to co-exist in a certificate (See <xref target="v1_coexistence"/>).
+          </t>
+        </section>
+        <section title="Verifying the Redacted SAN extension" anchor="verifying_redacted_san">
+          <t>
+            If the Redacted SAN extension is present, TLS clients MUST check that the SAN extension is present, that the SAN extension contains the same number of identities as the Redacted SAN extension, and that each identity in the SAN extension has a matching identity at the same position in the Redacted SAN extension. Two identities are matching if either:
+            <list style="symbols">
+              <t>The two identities are identical; or,</t>
+              <t>Both identities are DNS names, have the same number of labels, and each label in the SAN identity has a matching label at the same position in the Redacted SAN identity. Two DNS labels are matching if either:
+                <list style="symbols">
+                  <t>The two labels are identical; or,</t>
+                  <t>The label from the the Redacted SAN identity is equal to REDACTED(label from SAN identity) (<xref target="redacting_labels"/>)</t>
+                </list>
               </t>
             </list>
           </t>
           <t>
-            A certificate with redacted labels where one of the redacted labels is <spanx style="verb">*</spanx> MUST NOT be considered compliant.
+            If any of these checks fail, the certificate MUST NOT be considered compliant.
           </t>
           <t>
-            If the SCT checked is for a Precertificate (where the <spanx style="verb">type</spanx> of the <spanx style="verb">TransItem</spanx> is <spanx style="verb">precert_sct_v2</spanx>), then the client SHOULD also remove embedded v1 SCTs, identified by OID 1.3.6.1.4.1.11129.2.4.2 (See Section 3.3. of <xref target="RFC6962"/>), in the process of reconstructing the TBSCertificate. That is to allow embedded v1 and v2 SCTs to co-exist in a certificate (See <xref target="v1_coexistence"/>).
+            A certificate with redacted labels where one of the redacted labels is <spanx style="verb">*</spanx> MUST NOT be considered compliant.
```

Comment:
Would be clearer if you said something like a * in a SAN must also be a * (not redacted) in the redacted extension.